packages/web/src/hooks/common/use-clear-modal.tsx

```diff
@@ -1,11 +1,12 @@
 import { useAtom } from "jotai";
 import { CommonState } from "@states/index";
 import { useCallback } from "react";
+import { usePreventScroll } from "./use-prevent-scroll";
 
 export const useClearModal = () => {
-  const [, setOpenedModal] = useAtom(CommonState.openedModal);
+  const [_, setOpenedModal] = useAtom(CommonState.openedModal);
   const [, setModalContent] = useAtom(CommonState.modalContent);
-
+  usePreventScroll(_);
```

Comment:
What does it mean?

Reply:
I refactor it in the method container. This is just to prevent the content from scrolling when displaying Modal